randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -278,8 +278,11 @@ Extra - asset_id: 6
       Any of the following:
           All of the following:
               Gravity Suit
-              # With Wall Jumps: https://youtu.be/6tyDDiNLY_o
-              Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+              Any of the following:
+                  # With Wall Jumps: https://youtu.be/6tyDDiNLY_o
+                  Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+                  # 2 Tap Shinespark: https://youtu.be/a8-2ZAwcMos
+                  Speed Booster and Shinespark (Beginner) and Short Charge (Intermediate)
```

Comment:
>"2 tap"
>"beginner"

That does not add up.

Reply:
oh my bad, i didnt see the short charge req.

Reply:
The shine spark is beginner, the short charge is intermediate. Is that alright?

Reply:
yeah was just blind